src/views/CVERecord/CVERecord.vue

```diff
@@ -1,72 +1,87 @@
-          <div class="content">
-            <div class="cve-loading-message has-text-centered" v-if="this.$store.state.isSearching">
-              <span class="icon-text">
-                <span class="icon">
-                  <span class="loader is-loading"/>
+    <div v-if="usecveRecordStore.isSearching">
+      <div class="columns">
+        <div class="column is-2 is-hidden-touch"></div>
+        <div class="column is-full-mobile is-full-tablet is-7-desktop">
+          <main id="cve-main-page-content">
+            <div class="content">
+              <div class="cve-loading-message has-text-centered">
+                <span class="icon-text">
+                  <span class="icon">
+                    <span class="loader is-loading"/>
+                  </span>
+                  <span>Please wait. Loading...</span>
                 </span>
-                <span>Please wait. Loading...</span>
-              </span>
-            </div>
-            <div v-else>
-              <div v-if="this.$store.state.isIdOrRecordFound">
-                <PublishedRecord :NVDBaseUrl="NVDBaseUrl" v-if="this.$store.state.isPublished"/>
-                <RejectedRecordOrId :NVDBaseUrl="NVDBaseUrl" v-if="this.$store.state.isRejected"/>
-                <ReservedId v-if="this.$store.state.isReserved"/>
               </div>
-              <p v-if="this.$store.state.showHelpText" class="has-text-centered">
-                Please use the search box above to find a CVE record by ID.
-              </p>
-              <div v-if="this.$store.state.serverError" class="has-text-centered">
-                <h1 class="title is-4">Service is currently unavailable.</h1>
-                <p style="text-align: center;">Please
-                  <span class="icon-text">
-                    <a href="https://cveform.mitre.org/" target="_blank">report the issue
+            </div>
+          </main>
+        </div>
+      </div>
+    </div>
+    <div v-else>
+      <div v-if="usecveRecordStore.isIdOrRecordFound">
+        <PublishedRecord v-if="usecveRecordStore.isPublished"/>
+        <RejectedRecordOrId v-if="usecveRecordStore.isRejected"/>
+        <ReservedId v-if="usecveRecordStore.isReserved"/>
+      </div>
+      <div v-else class="columns">
+        <div class="column is-2 is-hidden-touch"></div>
+        <div class="column is-full-mobile is-full-tablet is-7-desktop">
+          <main id="cve-main-page-content">
+            <div class="content">
+              <div>
+                <p v-if="usecveRecordStore.showHelpText" class="has-text-centered">
+                  Please use the search box above to find a CVE record by ID.
+                </p>
+                <div v-else-if="usecveRecordStore.serverError" class="has-text-centered">
+                  <h1 class="title is-4">Service is currently unavailable.</h1>
+                  <p style="text-align: center;">Please
+                    <span class="icon-text">
```

Comment:
I think this is the same code block as `SearchResults.vue`, the `text-align` style probably doesn't affect the text. And if it does, I think the Bulma class would be better.

That being said, this can probably become a "ServiceUnavailable" component or something similar